Packs/FeedGCPWhitelist/Integrations/FeedGCPWhitelist/FeedGCPWhitelist.py

```diff
@@ -0,0 +1,153 @@
+from CommonServerPython import *
+
+# IMPORTS
+import dns.resolver
+import re
+from typing import List, Dict, Callable, Tuple, Any
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+# CONSTANTS
+GOOGLE_BASE_DNS = "_cloud-netblocks.googleusercontent.com"
+
+
+def fetch_cidr(dns_address: str) -> List[Dict]:
+    """Recursively builds a CIDR dictionary with the relevant ip and type
+
+    Args:
+        dns_address: the dns address to lookup
+
+    Returns:
+        CIDR list
+    """
+    cidr_arr = []
+    regex_dns = r"(include:.*? )"
+    regex_cidr = r"(ip.*?:.*? )"
+
+    query_response_str = str(dns.resolver.query(dns_address, "TXT").response.answer[0][0])
```

Comment:
Are these array indices always guaranteed to be there? If you are unsure, please add a `try` `except` block to catch an `IndexError`.